extensions-core/kafka-indexing-service/src/main/java/org/apache/druid/indexing/kafka/supervisor/KafkaSupervisor.java

```diff
@@ -1776,6 +1774,10 @@ private void checkCurrentTaskState() throws ExecutionException, InterruptedExcep
 
   void createNewTasks() throws JsonProcessingException
   {
+    if (spec.isSuspended()) {
```

Comment:
Is this needed, since `createNewTasks()` is only called when not suspended?

Reply:
I don't think so, I'll remove.